moj-be/modules/custom/moj_resources/config/install/rest.resource.you_might_like_resource.yml

```diff
@@ -0,0 +1,9 @@
+id: you_might_like_resource
```

Comment:
Can we rename it to suggestions? Across the board?

Reply:
Lol, yeah, I was going to ask suggest it

Reply:
@romidane sorted!